apps/web/vibes/soul/primitives/product-card/index.tsx

```diff
@@ -164,19 +165,17 @@ export function ProductCard({
 export function ProductCardSkeleton({ className }: { className?: string }) {
   return (
     <div className={className}>
-      <div className="flex aspect-[5/6] flex-col gap-2 rounded-xl bg-contrast-100 @md:rounded-2xl" />
+      <Skeleton
+        aspectRatio="5:6"
+        childClassName="rounded-xl w-full h-full @md:rounded-2xl"
```

Comment:
This seems weird. Do we want an API that allows us to pass the class directly to the element?